### DISCUSSION TITLE
Using TailwindCSS in Content.ts

### DISCUSSION THREAD

**Discussion by drigoalexander** [wxt-dev/wxt#1318] [Q&A]
How can we using TailwindCSS in content.ts for the injected content to the DOM?

**Comment by aklinker1** [comment]
Search for tailwind in WXT's examples: https://wxt.dev/examples.html

**Reply by curiosbasant** [reply]
How can I include the `tailwindcss@v4` css file in all content scripts?
Right now, I have a content script that matches `*//:**`, I am included the css file here, so it can be available on all scripts as well.
It kinda works, but I have now problem with `hmr` sometimes the new classes gets added and sometimes not.

So whats the best way to include the tailwind css file in all content scripts?

**Reply by oyzamil** [reply]
@curiosbasant , please share your project file so that we can assist you.
